### PR TITLE
fix broken unit test due to changed loadtest inputs

### PR DESCRIPTION
**Pull request by gatsbyz**

### CODE REVIEW

**Comment by Stefan-Ethernal** [comment]
@gatsbyz consider returning back Slack notifications for Nightly builds through this PR, because those were deleted accidentally I suppose through #1430 (https://github.com/0xPolygon/polygon-edge/commit/024d01104a311386ad7d23ed03f361ba987029de#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6L46-L49).

**Comment by gatsbyz** [comment]
@Stefan-Ethernal that was a duplicated part of the code. i think it was a rebase error on my part. so the notification block actually starts  https://github.com/0xPolygon/polygon-edge/commit/024d01104a311386ad7d23ed03f361ba987029de#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6L67-L71 here

**Review by vcastellm — APPROVED** [review]
LGTM

File: .github/workflows/manual.deploy.nightly.devnet.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      AWS_REGION: ${{ secrets.AWS_REGION }}
+      AWS_ROLE_ARN: ${{ secrets.AWS_ROLE_ARN }}
+      TF_VAR_DEPLOYMENT_NAME: ${{ secrets.TF_VAR_DEPLOYMENT_NAME }}
+      TF_VAR_OWNER: ${{ secrets.TF_VAR_OWNER }}
+      SLACK_PERFORMANCE_WEBHOOK_URL: ${{ secrets.SLACK_PERFORMANCE_WEBHOOK_URL }}
+      DD_API_KEY: ${{ secrets.DD_API_KEY }}
+      LOADTEST_MNEMONIC: ${{ secrets.LOADTEST_MNEMONIC }}
+      VAULT_PASSWORD_FILE: ${{ secrets.VAULT_PASSWORD_FILE }}
+    with:
+      environment: devnet
```
Comment (vcastellm) [inline .github/workflows/manual.deploy.nightly.devnet.yml]:
```suggestion
      environment: devnet

```

**Comment by Stefan-Ethernal** [comment]
> @Stefan-Ethernal that was a duplicated part of the code. i think it was a rebase error on my part. so the notification block actually starts [024d011#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6L67-L71](https://github.com/0xPolygon/polygon-edge/commit/024d01104a311386ad7d23ed03f361ba987029de#diff-0d5658b415099a82c11c03a06ca4ec765b4003a1f4b2f3f1943980a882cf8aa6L67-L71) here

Ok, but strange there weren't Slack notifications for yesterday's NB. :thinking:  There is an error in `nightly` workflow (https://github.com/0xPolygon/polygon-edge/actions/runs/4803384951), so it is fails when triggered:

> [Invalid workflow file: .github/workflows/nightly.yml#L52](https://github.com/0xPolygon/polygon-edge/actions/runs/4803384951/workflow)
> The workflow is not valid. .github/workflows/nightly.yml (Line: 52, Col: 28): Invalid secret, TF_VAR_COMPANY_NAME is not defined in the referenced workflow.

**Review by Nemanja0x — APPROVED** [review]

**Pull request merged**